### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by luc-boussant**

### CODE REVIEW

**Comment by DavidVujic** [comment]
Thank you for this Pull Request! I’ll have a look at it before the weekend.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+If applicable, add the error logs.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
```
Comment (DavidVujic) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Could you modify the the items for this header in the templae to fit the project better? Something like:
- OS:
- Node.js version:
- C/C++ compiler and version:

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
```
Comment (DavidVujic) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Could you modify the steps to fit the steps done in a terminal window? Maybe something like "open a terminal window", "type npm install" ...

**Comment by DavidVujic** [comment]
@luc-boussant Please have a look at the feedback above for your pull request. Can you modify the file according to the feedback? So I can merge it to the Master branch.

**Comment by DavidVujic** [comment]
@luc-boussant I think I have to close this pull request if no action is taken, to avoid this issue to be blocked. If you are interested in having this pull request merged, please have a look at the feedback!

**Comment by luc-boussant** [comment]
@DavidVujic  Sorry for the delay. I worked on your feedbacks. Tell me if this is good for you.

**Comment by DavidVujic** [comment]
Great! Thank you @luc-boussant for contributing to this project! The PR is now merged.

**Pull request merged**